core/src/main/scala/io/finch/internal/Mapper.scala

```diff
@@ -106,11 +181,37 @@ object Mapper extends HighPriorityMapperConversions {
     ev: FOB <:< F[Output[B]]
   ): Mapper.Aux[F, A, B] = instance(_.mapOutputAsync(value => ev(ftp(f)(value))))
 
-  implicit def mapperFromFutureResponseHFunction[F[_] : Monad, A, FN, FR](f: FN)(implicit
+  implicit def mapperFromEffectResponseHFunction[F[_] : Monad, A, FN, FR](f: FN)(implicit
     ftp: FnToProduct.Aux[FN, A => FR],
     ev: FR <:< F[Response]
   ): Mapper.Aux[F, A, Response] = instance(_.mapOutputAsync { value =>
     val fr = ev(ftp(f)(value))
     fr.map(r => Output.payload(r, r.status))
   })
+
+  implicit def mapperFromFutureOutputHFunction[F[_] : Effect, A, B, FN, FOB](f: FN)(implicit
+    ftp: FnToProduct.Aux[FN, A => FOB],
+    ev: FOB <:< TwitterFuture[Output[B]]
+  ): Mapper.Aux[F, A, B] = instance(_.mapOutputAsync(value => twitterFutureToEffect(ev(ftp(f)(value)))))
+
+  implicit def mapperFromFutureResponseHFunction[F[_] : Effect, A, FN, FR](f: FN)(implicit
+    ftp: FnToProduct.Aux[FN, A => FR],
+    ev: FR <:< TwitterFuture[Response]
+  ): Mapper.Aux[F, A, Response] = instance(_.mapOutputAsync { value =>
+    val fr = twitterFutureToEffect(ev(ftp(f)(value)))
+    fr.map(r => Output.payload(r, r.status))
+  })
+
+  implicit def mapperFromScFutureOutputHFunction[F[_] : Effect, A, B, FN, FOB](f: FN)(implicit
+                                                                                    ftp: FnToProduct.Aux[FN, A => FOB],
+                                                                                    ev: FOB <:< ScalaFuture[Output[B]]
+  ): Mapper.Aux[F, A, B] = instance(_.mapOutputAsync(value => scalaFutureToEffect(ev(ftp(f)(value)))))
+
+  implicit def mapperFromScFutureResponseHFunction[F[_] : Effect, A, FN, FR](f: FN)(implicit
+                                                                                  ftp: FnToProduct.Aux[FN, A => FR],
```

Comment:
Here as well.

Reply:
I hope we gonna have scalafmt soon 😞

Reply:
Yea that would be rad. I've seen cats recently adopted it too. Sounds like we're going to be a good company after all.